src/pages/verify/integrations/react-native-expo.mdx

```diff
@@ -0,0 +1,175 @@
+---
+product: verify
+category: Integrations
+sort: 1
+title: Expo (React Native)
+subtitle: Accept MitID, NemID, Swedish BankID, Norwegian BankID and more eID logins in your Expo (React Native) app with @criipto/verify-expo.
+---
+
+import Layout from '../../../layouts/mdx';
+export default Layout;
+
+Read the [full documentation](https://github.com/criipto/criipto-verify-expo) or [download a sample application from GitHub](https://github.com/criipto/criipto-verify-expo/tree/master/example).
+
+## Register Your Application in Criipto Verify
+
+After you signed up for Criipto Verify, you must [register an application](https://dashboard.criipto.com/applications/add) before you can try logging in with any eID.
+
+Once your application is registered, the dashboard will contain the necessary details to configure your application to communicate with Criipto Verify.
+You will need the following information from the General tab of your application settings:
+
+- _Client ID_ to identify your application to Criipto Verify.
+- _Domain_ on which you will be communicating with Criipto Verify.
+
+### Register callback URLs
+
+Before you can start sending authentication requests to Criipto Verify, you need to register the URLs on which you want to receive the returned _JSON Web Token_, [JWT](verify/getting-started/token-contents/).
+
+The callback URL of your application is the URL where Criipto Verify will redirect to after the user has authenticated in order for the OpenID Connect middleware to complete the authentication process.
+
+You will need to add this URL to the list of allowed URLs for your application.
+
+For example, if you are developing your application in Expo Go, the callback URL may look like this: `exp://127.0.0.1:8081`.
```

Comment:
Expand this sentence with "or if you've setup a custom scheme, it might look like `yourscheme://auth/callback`"